src/components/projects/view/application-dialog.svelte

```diff
@@ -22,6 +22,10 @@
       remember: remember[0].checked,
     });
     comment = '';
+    if (!remember[0].checked) {
+      telegram = savedUsername || '';
+    }
+    remember[0].checked = false;
```

Comment:
Why forget whether or not they want to **remember**?)

Reply:
If they remembered it once, no need to send subsequent saving requests, it's already saved

Reply:
But is that obvious to the user, or will they click "remember" on every application?

Reply:
It is, since the field will be pre-filled on subsequent applications (and not on the first one)